### PR TITLE
Add Metadata Required For Onboarding Onfido UI into Console and Management Console

### PR DESCRIPTION
**Pull request by nipunsampath**
## Purpose
$subject

### Configuration Properties
image

## Related PRs

- https://github.com/wso2-extensions/identity-verification/pull/9

### CODE REVIEW

File: ui-metadata/onfido/metadata.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "common": {
+      "configProperties": [
+        {
+          "name": "token",
+          "displayOrder": 1,
+          "hint": "The API token generated via the Onfido dashboard",
```
Comment (divyaamunugama) [inline ui-metadata/onfido/metadata.json]:
```suggestion
          "hint": "The API token generated via the Onfido dashboard.",
```

File: ui-metadata/onfido/metadata.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          "type": "password",
+          "required": true,
+          "isSecret": true,
+          "dataComponentId": "idvp-edit-config-settings-token",
+          "maxLength": 100
+        },
+        {
+          "name": "webhook_token",
+          "displayOrder": 2,
+          "hint": "The Webhook token generated via the Onfido dashboard",
```
Comment (divyaamunugama) [inline ui-metadata/onfido/metadata.json]:
```suggestion
          "hint": "The webhook token generated via the Onfido dashboard.",
```

Comment (divyaamunugama) [reply ui-metadata/onfido/metadata.json:18]: Add full stops to each hint.

File: ui-metadata/onfido/metadata.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          "type": "password",
+          "required": false,
+          "isSecret": true,
+          "dataComponentId": "idvp-edit-config-settings-webhook-token",
+          "maxLength": 100
+        },
+        {
+          "name": "base_url",
+          "displayOrder": 3,
+          "hint": "The base url of Onfido server",
```
Comment (divyaamunugama) [inline ui-metadata/onfido/metadata.json]:
```suggestion
          "hint": "The base URL of Onfido server.",
```

File: ui-metadata/onfido/metadata.json (hunk 1 above)
Comment (nipunsampath) [reply ui-metadata/onfido/metadata.json]: Resolved by 188a2b52e5a9ce9c7c137420c2c5d43f527ed7f4

File: ui-metadata/onfido/metadata.json (hunk 2 above)
Comment (nipunsampath) [reply ui-metadata/onfido/metadata.json]: Resolved by 188a2b52e5a9ce9c7c137420c2c5d43f527ed7f4

File: ui-metadata/onfido/metadata.json (hunk 3 above)
Comment (nipunsampath) [reply ui-metadata/onfido/metadata.json]: Resolved by 188a2b52e5a9ce9c7c137420c2c5d43f527ed7f4

**Review by divyaamunugama — APPROVED** [review]

**Review by GANGANI — APPROVED** [review]

**Pull request merged**